src/components/Users/InvitesTable.js

```diff
@@ -37,20 +37,20 @@ const getTableColumnsConfig = () => {
     {
       classes: 'created',
       dataField: 'created',
-      text: 'INVITED',
+      text: 'Invited',
```

Comment:
Not within the scope of this PR, but after removing `_table.sass`, `react-bootstrap-table-next` tables no longer have styled headers.

Not a big deal for now, but until the `smart` table refactor.

<details>
<summary>Before</summary>

![image](https://user-images.githubusercontent.com/13508038/109325679-82af9e00-7856-11eb-8d22-fd441f0deebe.png)

</details>

<details>
<summary>Now</summary>

![image](https://user-images.githubusercontent.com/13508038/109325734-922ee700-7856-11eb-8b82-2b1de3830a58.png)

</details>

Reply:
Looks ok, but kinda weird that we go back and forth on styles. Can we set a global style somewhere? Or instead just take this as our new style?

Reply:
This style is just for `react-bootstrap-table-next` tables, and is temporary, until I refactor those, as well (those have to use the more complex `DataTable` `grommet` component)